wgengine/magicsock/derp.go

```diff
@@ -732,6 +731,9 @@ func (c *Conn) processDERPReadResult(dm derpReadResult, b []byte) (n int, ep *en
 	if stats := c.stats.Load(); stats != nil {
 		stats.UpdateRxPhysical(ep.nodeAddr, ipp, 1, dm.n)
 	}
+
+	c.metrics.inboundPacketsDERPTotal.Add(1)
+	c.metrics.inboundBytesDERPTotal.Add(int64(n))
```

Comment:
I moved these from `receiveDERP` here to make it more clear that these are incremented after `handleDiscoMessage`, and to keep them close to the stats counters.